slirp4netns.c

```diff
@@ -81,6 +89,13 @@ int do_slirp(int tapfd, int exitfd, unsigned int mtu, const char *api_socket, bo
 		g_array_append_val(&pollfds, exit_pollfd);
 		pollfds_exitfd_idx = n_fds - 1;
 	}
+	if (hostfwd_count > 0) {
+		for (int i=0; i < hostfwd_count; i++) {
+			hostfwd_t f = forwards[i];
+			if (slirp_add_hostfwd(slirp, f.is_udp, f.hostaddr, f.hostport, f.guestaddr, f.guestport) < 0)
```

Comment:
Please allow `api.c` to list and remove the hostfwd

Reply:
Easiest way to do this would be to extract these lines, in an own function correct? (From api_handle_req_add_hostfwd)
        fwd->id = ctx->hostfwds_nextid;
        ctx->hostfwds_nextid++;
        QTAILQ_INSERT_TAIL(&ctx->hostfwds, fwd, entry);